dotnet-test-samples/apigw-lambda-ddb/src/GetProduct/.lambda-test-tool/SavedRequests/API.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Should this file be ignored? It looks autogenerated and not customized to the project